extensions/http-curl/tests/CivetStream.h

```diff
@@ -119,7 +119,7 @@ class CivetStream : public io::BaseStream {
     return readData(reinterpret_cast<uint8_t *>(&buf[0]), sizeof(t));
   }
 
-  //size_t pos;
+  // size_t pos;
```

Comment:
some of these should probably be deleted rather than fixing the whitespace in them; this is an example of that

Reply:
Deleted.